flow/record/fieldtypes/__init__.py

```diff
@@ -5,13 +5,14 @@
 import os
 import pathlib
 import re
+import shlex
 import sys
 import warnings
 from binascii import a2b_hex, b2a_hex
 from datetime import datetime as _dt
 from datetime import timezone
 from posixpath import basename, dirname
-from typing import Any, Optional, Tuple
+from typing import Any, List, Optional, Tuple
```

Comment:
```suggestion
from typing import Any, Optional
```
Can you change the instances of `List` to `list` and `Tuple` to `tuple`?